# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-viewport/zoom/widget.html
# Commit: 4b595e1843aa
# Full Hash: 4b595e1843aa64b05f111f363838f3adea239f3a
# Author: Emilio Cobos Álvarez <emilio@crisal.io>
# Date: 2024-09-19 21:27:30
# Regressor Bug: 1918838
# File Overlap Count: 1
# Description:
#   Bug 1918838 - Make CSS zoom apply to themed widgets properly. r=dshin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D222320
# ==============================================================================

diff -r 68a97b80a5b7 -r 4b595e1843aa testing/web-platform/tests/css/css-viewport/zoom/widget.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-viewport/zoom/widget.html	Thu Sep 19 09:56:50 2024 +0000
@@ -0,0 +1,19 @@
+<!doctype html>
+<link rel="help" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1918838">
+<link rel="help" href="https://drafts.csswg.org/css-viewport/#zoom-property">
+<link rel="author" href="mailto:emilio@crisal.io" title="Emilio Cobos Álvarez">
+<link rel="author" href="https://mozilla.org" title="Mozilla">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<input type=checkbox>
+<input type=radio>
+<script>
+for (let i of document.querySelectorAll("input")) {
+  test(function() {
+    let unzoomed = i.getBoundingClientRect().width;
+    i.style.zoom = "2";
+    let zoomed = i.getBoundingClientRect().width;
+    assert_not_equals(unzoomed, zoomed, "Zoom should be applied");
+  }, i.type);
+}
+</script>